docs/tutorial.md

```diff
@@ -32,6 +32,8 @@ The `init` command generates the RBAC rules in `config/rbac/role.yaml` based on
 **Note:**
 This creates a project structure that is compatible with both Helm and Go APIs. To learn more about the project directory structure, see the [project layout][[project_layout]] doc.
 
+Hybrid-Helm has not been tested against webhooks and cert manager. Hencem there is no scaffolding for webhooks and cert manager.
```

Comment:
Can we also add that, Go APIs support Webhooks in general. If users, would like to experiment doing so, they can refer to SDK(or KB) documentation, and add the required scaffolding, **but for only Go APIs**. Can we please add the link to respective SDK/KB doc that talks about webhooks for Go types.

Reply:
Updated doc.